ipn/ipnstate/ipnstate.go

```diff
@@ -108,8 +108,12 @@ type PeerStatus struct {
 	OS        string // HostInfo.OS
 	UserID    tailcfg.UserID
 
+	Tags []string // Tags assigned to this node
```

Comment:
omitempty?

And maybe promote to a lines-before formal comment and describe this more: is this only the ones that were accepted or the ones that were requested?

Reply:
Done, also added a pointer to tailcfg#Node.Tags.